source/fundamentals/bson/undefined-values.txt

```diff
@@ -0,0 +1,119 @@
+.. _node-undefined-values:
+
+================
+Undefined Values
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to control how the driver serializes
+undefined values. By default, the driver serializes ``undefined`` values
+as ``null`` values during write operations.
+
+.. _nodejs-specify-ignoreundefined:
+
+Specify the ``ignoreUndefined`` Setting
```

Comment:
Suggestion: I think this section could potentially be renamed something like "Ignore Undefined Values" since that's the core of what they are learning how to do here.